app/models/manageiq/providers/ibm_cloud/vpc/network_manager/cloud_subnet.rb

```diff
@@ -14,6 +15,49 @@ class ManageIQ::Providers::IbmCloud::VPC::NetworkManager::CloudSubnet < ::CloudS
     end
   end
 
+  def self.params_for_create(ems)
+    {
+      :fields => [
+        {
+          :component    => 'select',
+          :name         => 'cloud_network_id',
+          :id           => 'cloud_network_id',
+          :label        => _('Network'),
+          :isRequired   => true,
+          :includeEmpty => true,
+          :validate     => [{:type => 'required'}],
+          :options      => ems.cloud_networks.map do |cvt|
+            {
+              :label => cvt.name,
+              :value => cvt.id,
+            }
+          end
+        }
+      ]
+    }
+  end
+
+  def self.raw_create_cloud_subnet(ext_management_system, options)
+    cloud_network_id = options.delete(:cloud_network_id)
+    cloud_network = CloudNetwork.find_by(:id => cloud_network_id) if cloud_network_id
+    options[:network_id] = cloud_network&.ems_ref
```

Comment:
Similarly, you don't have to set this back in options, you could just say `subnet = {:vpc => {:id => cloud_network&.ems_ref}}` below